Apps/WebClient/src/appsettings.json

```diff
@@ -49,5 +49,9 @@
       "Encounter": "false",
       "Note": "false"
     }
+  },
+  "AvailabilityFilter": {

```

Comment:
We should merge this and the modules to have a single point of configuration for enabling/disabling modles (medications. imz, etc)

Reply:
I'll raise a task for this as the refactoring may not be worth it.  Currently External Configuration is loaded completely from WebClient but this Filter should be at the root level as it applies to all applications.